perfkitbenchmarker/benchmark_spec.py

```diff
@@ -321,8 +321,11 @@ def PrepareVm(self, vm, firewall):
     vm.Create()
     logging.info('VM: %s', vm.ip_address)
     logging.info('Waiting for boot completion.')
-    for port in vm.remote_access_ports:
-      firewall.AllowPort(vm, port)
+    # If the user has requested to use a specific subnet ID, that has already
+    # been setup with allowed port, we cannot invoke it again here.
+    if FLAGS.existing_subnet_id is None:
```

Comment:
I don't like checking a cloud specific flag in a file shared by many clouds - can you pull it up into aws_network.py?